src/views/app/detail-panel/edit/edit-view.tsx

```diff
@@ -245,20 +245,39 @@ export const EditView = React.forwardRef<EditViewHandle, EditViewProp>(function
 		[createSnackbar, editorId]
 	);
 
-	const onSendError = useCallback((): void => {
-		createSnackbar({
-			key: `mail-${editorId}`,
-			replace: true,
-			severity: 'error',
-			label: t('label.error_try_again', 'Something went wrong, please try again'),
-			autoHideTimeout: TIMEOUTS.SNACKBAR_DEFAULT_TIMEOUT,
-			hideButton: true
-		});
-		createEditBoard({
-			action: EditViewActions.RESUME,
-			actionTargetId: editorId
-		});
-	}, [createSnackbar, editorId]);
+	function isErrorAboutInvalidRecipient(error?: unknown): boolean {
+		return typeof error === 'string' && error.includes('5.1.1');
```

Comment:
@FrancescoGottardi I am not sure we should rely on this magic string `5.1.1`.

It might be more reliable to check the error code (`SEND_ABORTED_ADDRESS_FAILURE`) instead. 

What do you think?

Reply:
We saw that the 5.1.1 code is an LMTP code, so it might be better to rely on the SOAP status code, as the 5.1.1 appears to be an internal error